wrapper/src/main/java/software/amazon/jdbc/plugin/failover2/FailoverConnectionPlugin.java

```diff
@@ -483,8 +483,14 @@ protected void failoverWriter() throws SQLException {
           .findFirst()
           .orElse(null);
 
+      if (writerCandidate == null) {
+        this.failoverWriterFailedCounter.inc();
+        LOGGER.severe(Utils.logTopology(updatedHosts, Messages.get("Failover.noWriterHost")));
+        throw new FailoverFailedSQLException(Messages.get("Failover.unableToConnectToWriter"));
```

Comment:
Should we use this message instead? 
Messages.get("Failover.noWriterHost")

Reply:
Sure, I updated the other FailoverFailedSQLException messages to match the logs I added as well so that they are more specific.